### DISCUSSION TITLE
One can dream...

### DISCUSSION THREAD

**Discussion by joelmayer87** [keiwando/toonsquid#696] [General]
... of ToonSquid on macOS :D 

<img width="641" alt="image" src="https://github.com/user-attachments/assets/5706fd00-da45-4a00-bac4-b8fc3bf15132">

(no actually running software, i'm just mirroring my iPad to my Mac but wanted to see how it would feel like :P )

PS: 2.0 is so good it should be on desktop, i'd seriously pay 10-20 times for it.

**Comment by DWrightAnimation** [comment]
You shared a screenshot of 2.0, does that mean the NDA on sharing anything about it has expired, and it might be coming soon? Could you also share how the new features work, like the FX? Also, from what I understand, MacOS isn't as friendly with open source software as Windows, so running OpenToonz on it might not be ideal, so a MacOS compatible version of ToonSquid would entice me to buy a Mac, whether it's a MacBook, Mini, or anything.

**Reply by joelmayer87** [reply]
Oh no, you're right, i thought i had the 1.x version open, ooops my bad sorry about that. Luckily didn't really give anything away... won't happen again!

No, as long as Toonsquid is not released to the public we won't say anything, that's up to the developer. Except that it's really good ;) No idea about release date.

**Reply by DWrightAnimation** [reply]
> Oh no, you're right, i thought i had the 1.x version open, ooops my bad sorry about that. Luckily didn't really give anything away... won't happen again!
> 
> 
> 
> No, as long as Toonsquid is not released to the public we won't say anything, that's up to the developer. Except that it's really good ;) No idea about release date.

Well, I'll say I appreciate at least having one glimpse at 2.0 to tide me over for a while.

**Comment by budaloops** [comment]
When I first saw the title of this post, I thought it had something to do with Procreate Dreams’ 1.1 update (with the lasso tool) taking forever to come out compared to Toonsquid’s regular updates 🙈

**Reply by joelmayer87** [reply]
Toonsquid makes all your Dreams come true ;)

**Comment by Cubishon** [comment]
Hi Joel,
I’m curious—why do you need it on a computer? Is it because of the larger screen, or is it about using programs that are only available on laptops, like being able to switch faster between ToonSquid and After Effects, for example?

**Reply by joelmayer87** [reply]
Yes, larger screen, better file management, faster to go from animation to comp and edit, more keyboard shortcuts and workflow which i find still the speediest way to work (i use a little Huion KeyDial with Toonsquid which already helps but having shortcuts for other things would be awesome), etc. etc. :) 

And then imagine Toonsquid shares a file-type so you can effortlessly switch between iPad and Mac in the same project kinda like Clip Studio.
